phabfive/paste.py

```diff
@@ -5,15 +5,36 @@
 
 # phabfive imports
 from phabfive.core import Phabfive
+from phabfive.exceptions import PhabfiveDataException
 
 class Paste(Phabfive):
     def __init__(self):
         super(Paste, self).__init__()
 
-    def get_pastes(self):
+    def get_pastes(self, queryKey="all", attachments=None, constraints=None):
```

Comment:
Change `queryKey` to `query_key`. Internally we should use the pep8 naming standard and that says snakecase and not camelcase. Yet we can't change the upstream lib of phabricator that uses `queryKey`, but unless we monkeypatch of subclass the same method.

Reply:
@Grokzen if we do it that way then I would also propose adding a mapping at the top of such method to translate our snake_case into the matching camelCase. In practice that means every method that uses the Phabricator API would have to have a mapping.

Reply:
I would say that phabricator API should fix their code :) But that won't happen any time soon. But i do not get why we need another mapping inside here that maps the snake_case back to camelcase. This function is made in order to map the input variables from this method into the phabricator API call. Like it is already done on L20 in this MR. We can't change the `self.phab.paste.search` method signature itself, but we should have snake_case inside our own code.

Reply:
We will merge this as is. We will change the arguments to PEP-8 in one fell swoop instead, as there are more places that are affected.